### PR TITLE
libp2p, debugapi: add API to view blocklisted peers

### PR DESCRIPTION
**Pull request by acud**
adds a debugapi to view currently blocklisted peers. if a peer is in the blocklist but the duration has already expired, it will not show up in the endpoint. however, unlike in the `blocklist.Exists` method, the entry will not get deleted while querying this endpoint, in order to reduce side-effects.

ref https://github.com/ethersphere/bee/issues/1053

### CODE REVIEW

**Review by ralph-pichler — APPROVED** [review]

File: pkg/debugapi/router.go [hunk 1]
```diff
@@ -68,6 +68,10 @@ func (s *server) setupRouting() {
 	router.Handle("/peers", jsonhttp.MethodHandler{
 		"GET": http.HandlerFunc(s.peersHandler),
 	})
+	router.Handle("/peers/blocklisted", jsonhttp.MethodHandler{
```
Comment (zbiljic) [inline pkg/debugapi/router.go:71]: This endpoint clashes with `"/peers/{address}"` (just bellow it).
While they do use different method, and this is not public API, I still think some other than `"/peers/blocklisted"` should be used (maybe just reverse words). At least to keep the API "clean" and discourage similar practice for the future.

**Review by zbiljic — CHANGES_REQUESTED** [review]

File: pkg/debugapi/router.go (hunk 1 above)
Comment (acud) [reply pkg/debugapi/router.go]: Thanks @zbiljic. I renamed the endpoint to `/blocklist`

File: pkg/p2p/p2p.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ type Service interface {
 	Connect(ctx context.Context, addr ma.Multiaddr) (address *bzz.Address, err error)
 	Disconnecter
 	Peers() []Peer
+	BlocklistedPeers() ([]Peer, error)
```
Comment (zbiljic) [inline pkg/p2p/p2p.go:24]: It looks a bit strange that function for listing blocklisted peers returns an error, but list of peers does not.

**Review by zbiljic — APPROVED** [review]

File: pkg/p2p/p2p.go (hunk 2 above)
Comment (acud) [reply pkg/p2p/p2p.go:24]: true. but `Peers` does not involve any storage operations, therefore does not return a possible error :+1:

**Pull request merged**